app/controllers/project_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ProjectController < ApplicationController
```

Comment:
This is absolutely my bad as I should have spotted it earlier! By convention, Rails controllers are pluralised: 

https://guides.rubyonrails.org/action_controller_overview.html#controller-naming-convention

This singular project may bite us later so I think we should change it to`ProjectsController` the routes will need updating as well.

Apologies!